src/validators/contains.js

```diff
@@ -0,0 +1,19 @@
+const checker = require("../checker")
+const err = require("../errorCodes")
+
+function contains(value, options) {
+  let results = null
+  if (checker.isEmpty(value)) return null
+
+  if (options.allowed && !checker.contains(options.allowed, value)) {
+      results = results || []
+      results.push({ [err.invalidInclusion]: options.allowed })
+    }  
```

Comment:
is this the right space/tab?

we might need a prettier post-processor before submit code